docs/content/middlewares/http/ratelimit.md

```diff
@@ -28,11 +28,11 @@ labels:
 apiVersion: traefik.io/v1alpha1
 kind: Middleware
 metadata:
-  name: test-ratelimit
+    name: test-ratelimit
 spec:
-  rateLimit:
-    average: 100
-    burst: 200
+    rateLimit:
+        average: 100
+        burst: 200
```

Comment:
Do you know what happened with the indentation in this file? 
Could you keep it consistent with the rest of the documentation?